Source/WebCore/css/CSSFontFace.cpp

```diff
@@ -69,16 +69,17 @@ void CSSFontFace::appendSources(CSSFontFace& fontFace, CSSValueList& srcList, Sc
 
         foundSVGFont = item.isSVGFontFaceSrc() || item.svgFontFaceElement();
         fontFaceElement = item.svgFontFaceElement();
+        const auto* settings = context ? &context->settingsValues() : nullptr;
+        bool allowDownloading = settings && settings->downloadableBinaryFontsEnabled;
```

Comment:
I would have written this as:

bool allowDownloading = false;
if (context && context->settingsValues()->downloadableBinaryFontsEnabled)
    allowDownloading = true;

...because I always find ?: confusing, and this formulation avoids a local pointer variable of indeterminate lifetime.

Reply:
Wouldn't it be better to say
```
bool allowDownloading = context && context->settingsValues()->downloadableBinaryFontsEnabled;
```
?

Reply:
I like that too!